src/modules/endpoint/MemoryDatabase.js

```diff
@@ -184,29 +195,73 @@ class MemoryDatabase {
     }))
   }
 
-  async readResourceCache () {
+  readLanguages = async () => {
+    const path = this.getContentPath('languages')
+    const fileExists: boolean = await RNFetchblob.fs.exists(path)
+
+    if (!fileExists) {
+      this._languages = null
+      return
+    }
+
+    const languages = JSON.parse(await this.readFile(path))
+    this._languages = languages.map(language => new LanguageModel(language._code, language._name))
+  }
+
+  writeLanguages = async () => {
+    if (!this._languages) {
+      console.warn('MemoryDatabase does not have data to save!')
+      return
+    }
+    const path = this.getContentPath('languages')
+    await this.writeFile(path, JSON.stringify(this._languages))
+  }
+
+  readEvents = async () => {
+    const path = this.getContentPath('events')
+    const fileExists: boolean = await RNFetchblob.fs.exists(path)
+
+    if (!fileExists) {
+      this._events = null
+      return
+    }
+
+    this._events = JSON.parse(await this.readFile(path))
+  }
+
+  writeEvents = async () => {
+    if (!this._events) {
+      console.warn('MemoryDatabase does not have data to save!')
+      return
+    }
+    const path = this.getContentPath('events')
+    await this.writeFile(path, JSON.stringify(this._events))
+  }
+
+  readResourceCache = async () => {
```

Comment:
we should not use lambda functions as function declarations if we don't need them.
Instead do sth like
```js
async readResourceCache () {
```

Reply:
Done.